include/classes/BasicChecks.class.php

```diff
@@ -160,7 +160,7 @@ public static function getNumOfTagRecursiveInChildren($e, $tag)
 	*/
 	public static function hasDuplicateAttribute($e, $attr, $id_array)
 	{
-		global $has_duplicate_attribute, $id_array;
+		global $has_duplicate_attribute;
```

Comment:
I wonder why `global $id_array` is removed? I believe this declaration is to make sure all ids from recursive function calls at line 179 are tracked in it. Without this declaration, each recursive call will start a new local `$id_array` only with ids captured in this single call.

Reply:
I notice including that, do result to this error message being shown in both PHP 5.6 and 7
![errorbasiccheck](https://user-images.githubusercontent.com/15857679/42102307-6d805b2a-7bbd-11e8-8e7b-4b491302971f.png)